tox.ini

```diff
@@ -12,6 +12,10 @@ commands =
 	pydocstyle -v src/
 	py.test --cache-clear -vv src/
 
+[pytest]
+timeout = 10
+timeout-method = thread
```

Comment:
If you are not patching purest, why threading is needed?

Reply:
That is, infact, correct.